sql/api/src/main/scala/org/apache/spark/sql/streaming/StatefulProcessor.scala

```diff
@@ -50,9 +50,9 @@ trait StatefulProcessor[K, I, O] extends Serializable {
    *                    time if available
    * @return - Zero or more output rows
    */
-  def handleInputRow(
+  def handleInputRows(
```

Comment:
We are not yet enabling the ability of chaining, right? Then it's OK to go ahead. Probably need to file a JIRA ticket to deal with this.

Reply:
Created JIRA - https://issues.apache.org/jira/browse/SPARK-46854